### ISSUE TITLE
feature request: add CSS selector-like options to URL parameter tags

### ISSUE THREAD

**Issue by jmaeat** [maizzle/framework#749]
- Maizzle Version: 4.0.2
- Node.js Version: 18.1.0

Would be sweet if I could tailor the way URL parameters are applied to links like a CSS selector. I don't necessarily want to apply parameters to _all_ URLs in my templates, but would like to apply them only where the href contains `example.com`. 

It would be lush to do something like this in `config.js`:

```js
module.exports = {
  urlParameters: {
    _options: {
      tags: ['a[href~="example.com"]'],
      qs: {}
    },
    utm_source: 'maizzle',
    utm_medium: 'email',
  }
}
```

**Comment by cossssmin** [comment]
Should be possible, will play around with [`posthtml-match-helper`](https://github.com/posthtml/posthtml-match-helper) and see what we can do 👍

**Comment by cossssmin** [comment]
Added in [`posthtml-url-parameters@1.1.0`](https://github.com/posthtml/posthtml-url-parameters#tags), will be available in the next Maizzle feature release. Thanks!

**Comment by jmaeat** [comment]
You are amazing, thank you @cossssmin!

**Comment by cossssmin** [comment]
Released in 4.1.0, feel free to reopen if you spot any issues. Thanks!

https://github.com/maizzle/framework/releases/tag/v4.1.0
